README.md

```diff
@@ -26,6 +29,30 @@ In its current state, the swarm will:
 - Be received by the tickRecorder service
 - Write the data point into InfluxDB
 
+## Things that you can do with this setup
+### Micro Web Console
+
+http://localhost:8082
+
+### NATS
+
+Install nats-top
+```
+go get github.com/nats-io/nats-top
+```
+
+Run nats-top
+```
+NATS_IP=$(docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' niifinance_nats_1)
```

Comment:
Remove the NATS_IP line and replace the next line with DOCKER_IP, since its already declared above.